x/programs/rust/wasmlanche-sdk/src/types.rs

```diff
@@ -1,4 +1,5 @@
-use borsh::{BorshDeserialize, BorshSerialize};
+use borsh::BorshDeserialize;
+use serde::Serialize;
```

Comment:
I think you can do what you're looking for with `AsRef<[u8]>` instead of making `Address` `Serialize`.

Reply:
I'm not sure why, `Address` has to be serialized in the `actor` field of the `TestContext`

Reply:
You have written a manual implementation of `TestContext`, that's definitely better. Thanks!